browser-extension/src/components/AppRouter.tsx

```diff
@@ -11,56 +12,54 @@ import RegistrationPage from "./RegistrationPage";
 import SubscribersPage from "./SubscribersPage";
 import WalletPage from "./WalletPage";
 
-const AppRouter: FC = () => {
-  const { token, page } = useContext(AppContext)!;
+const AuthenticatedPages = [
+  Page.Profile,
+  Page.Wallet,
+  Page.Permissions,
+  Page.History,
+  Page.Subscribers,
+];
 
-  let pageContent: ReactElement;
+const AuthenticatedRouter: FC = () => {
+  const { page, setPage } = useContext(AppContext)!;
 
-  if (token) {
-    switch (page) {
-      case Page.Profile:
-        pageContent = <ProfilePage />;
-        break;
+  useEffect(() => {
+    if (!AuthenticatedPages.includes(page)) {
+      setPage(AuthenticatedPages[0]);
```

Comment:
i might be stupid but what is authPages[0], are we keeping track of only the pages that are visited after authentication

Reply:
![image](https://github.com/fydp-2024-55/fydp-2024.55/assets/49622234/0a38cd85-7aa3-4407-ad7c-ba152c87c080)
It's just a list of enum values.